src/proto/proto/db3_session.proto

```diff
@@ -0,0 +1,52 @@
+//
+//
+// db3_mutation.proto
+// Copyright (C) 2022 db3.network Author imotai <codego.me@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+import "db3_base.proto";
+package db3_session_proto;
+
+enum SessionStatus {
+    Running = 0;
+    Blocked = 1;
+    Stop = 2;
+}
+
+message QuerySessionInfo {
+    // the hex encoded string
+    int32 id = 1;
+    int64 start_time = 2;
+    SessionStatus status = 3;
+    int32 query_count = 4;
```

Comment:
add a total_query_weight field

Reply:
let's discuss it in https://github.com/dbpunk-labs/db3/issues/232 firstly and will implement it in the near furture.